### ISSUE TITLE
 [Bug]: The ratio of WeightedRanker in hybridsearch doesn't seem to be effective

### ISSUE THREAD

**Issue by manchiy** [milvus-io/milvus#34745]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:newest
- Deployment mode(standalone or cluster):standalone
- SDK version(e.g. pymilvus v2.0.0rc2): pymilvus 
- OS(Ubuntu or CentOS): Ubuntu
```

### Current Behavior

Collection A with two vectors , one vector is the original data vector, and the other vector is the vector that summarizes the data.
 Collection B with single vector, only the original data vector and the same as the vector of CollectionA  original data vector
 
 When searching for the same query, when I set the WeightedRanker weight of Collection A to [0.99,0.01], and set it to [0.8,0.2],[0.5,0.5], [0.2,0.8], [0.01,0.99], there is almost no difference in scores between 0.01.

The difference in search scores between Collection A [0.99,0.01] and Collection B is relatively large, exceeding 0.1. But theoretically, shouldn't the score difference between Collection A [0.99,0.01] and Collection B search be very small, because the weights are all in the original vector.

**Comment by yanliang567** [comment]
The score in hybridsearch is a reranking similarity, and it is not the same meaning as it in search. I don't think it is meaningful to compare them directly.
/assign @czs007 
any details to share for the difference?

/assign @manchiy

**Comment by xiaofan-luan** [comment]
can you share your scripts for this test.
I'm confused about what you are trying to do. All the milvus hybrid search only happened on one collection

**Comment by manchiy** [comment]
@xiaofan-luan @yanliang567 Sorry, I was busy with other things and didn't notice the reply. I am using this hybrid search on a collection. However, when performing vector search, it is essential to set a score to filter out irrelevant results, not just rely on top_K. I tested the above scenarios because I found it challenging to filter by score after using hybrid search. I suspect, as the bro above mentioned, that the scores are re-ranked rather than the original scores, making it difficult for me to set an effective score filter. As a result, the search results are always top_K, even if the content is irrelevant.
